.gitignore

```diff
@@ -12,3 +12,4 @@
 
 *.css
 *.css.map
+frontend/.env
```

Comment:
if you exclude the .env than add a .env.example to document the required variables for someone who clones this repo